conf/machine/verdin-imx8mm.conf

```diff
@@ -66,10 +66,13 @@ PREFERRED_PROVIDER_virtual/kernel = "linux-toradex"
 PREFERRED_PROVIDER_virtual/kernel_preempt-rt = "linux-toradex"
 PREFERRED_PROVIDER_virtual/bootloader = "u-boot-toradex"
 PREFERRED_PROVIDER_u-boot = "u-boot-toradex"
+
 # Currently we use the nxp 2020.04 downstream for mx8 as some features are
 # not yet available in 2020.07.
 PREFERRED_VERSION_u-boot-toradex = "2020.04%"
-
+# Updated gdk-pixbuf required for electron >=19
```

Comment:
small thing
`electron >=19` ->  `electron >=21` since I switched electron version from v19 to v21.

Reply:
That's true, but I think the limit is 19